docs/reference/commandline/info.md

```diff
@@ -40,64 +40,76 @@ available on the volume where `/var/lib/docker` is mounted.
 
 ### Show output
 
-The example below shows the output for a daemon running on Red Hat Enterprise Linux,
-using the `devicemapper` storage driver. As can be seen in the output, additional
-information about the `devicemapper` storage driver is shown:
+The example below shows the output for a daemon running on ubuntu Linux,
```

Comment:
Nit: ubuntu => Ubuntu

```suggestion
The example below shows the output for a daemon running on Ubuntu Linux,
```

Reply:
Ah, nice catch!

Actually, let me clean up this section; it's about `--debug`, but then continues to show "a machine with swarm enabled", which is quite irrelevant for this; we could just describe; additional information, such a the `Debug` section.

Reply:
@thaJeztah could you please let me know if you're fine with the changes. I can push them to branch myself if that works better. Thanks.